tempto-kafka/pom.xml

```diff
@@ -42,4 +42,15 @@
             <version>2.4.1</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <failIfNoTests>false</failIfNoTests>
```

Comment:
Why would you want to do that?
If there are no tests, we should exclude surefire run.

Reply:
There are no tests in these modules but after airbase update surefire will fail on lack of tests

Reply:
add this to properties instead:
```
 <!-- there are no tests -->
 <maven.test.skip>true</maven.test.skip>
```

Reply:
Airbase 117 will have a property for this

Reply:
Seems you can use `maven.test.skip` for now, but need to switch to `air.test.skip` after airbase 117, as airbase upgrade will render `maven.test.skip` ineffective. (This is a breaking change in airbase.)

Reply:
I've moved to airbase 117 to use `air.test.skip`